openid-4-verifiable-presentations-1_0.md

```diff
@@ -581,12 +581,24 @@ The following is a non-normative example of the Authorization Response that is s
     state=eyJhb...6-sVA
 ```
 
-If the Response Endpoint has successfully processed the request, it MUST respond with HTTPS status code 200. 
+The following is a non-normative example of an Authorization Error Response that is sent via an HTTPS request to the Verifier's Response Endpoint:
+
+```
+  POST /post HTTP/1.1
+    Host: client.example.org
+    Content-Type: application/x-www-form-urlencoded
+
+    error=...&
+    error_description=...&
+    state=eyJhb...6-sVA
+```
+
+If the Response Endpoint has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTPS status code 200.
 
 The following new parameter is defined for use in the response from the endpoint:
 
 `redirect_uri`:
-: OPTIONAL. When the redirect parameter is used the Wallet MUST send the User Agent to this redirect URI. The redirect URI allows the Verifier to continue the interaction with the End-User on the device where the Wallet resides after the Wallet has sent the Authorization Response to the Response URI. It especially enables the Verifier to prevent session fixation ((#session_fixation)) attacks.
+: OPTIONAL. Value is a string containing a URI. When this parameter is present the Wallet MUST direct the User Agent to this URI. This allows the Verifier to continue the interaction with the End-User on the device where the Wallet resides after the Wallet has sent the Authorization Response to the Response Endpoint. It can be used by the Verifier to prevent session fixation ((#session_fixation)) attacks. The Response Endpoint MAY return the `redirect_uri` parameter for successful Authorization Responses or for Error Responses.
```

Comment:
```suggestion
: OPTIONAL. A string containing a URI. When this parameter is present the Wallet MUST direct the User Agent to this URI. This allows the Verifier to continue the interaction with the End-User on the device where the Wallet resides after the Wallet has sent the Authorization Response to the Response Endpoint. It can be used by the Verifier to prevent session fixation ((#session_fixation)) attacks. The Response Endpoint MAY return the `redirect_uri` parameter for successful Authorization Responses or for Error Responses.
```